crates/holochain/src/conductor/handle.rs

```diff
@@ -412,14 +416,49 @@ impl<DS: DnaStore + 'static> ConductorHandleT for ConductorHandleImpl<DS> {
                     .map_err(holochain_p2p::HolochainP2pError::other);
                 respond.respond(Ok(async move { res }.boxed().into()));
             }
+            PutMetricDatum {
+                span_context,
+                respond,
+                dna_hash,
+                to_agent,
+                metric,
+            } => {
+                let env = { self.conductor.read().await.p2p_env(space) };
+                let agent = todo!("can we use to_agent?");
```

Comment:
I don't think to_agent is correct... that's used mainly to dispatch to cells--meaning it's the local destination agent for events.

But, this call, and all the other p2p calls are intercepted before they get to the cell, they really shouldn't even *have* a to_agent, and sometimes they're filled in with garbage data...

Also, the type is wrong, we need something like `remote_agent: Arc<KitsuneAgent>`.